src/ru/javarush/ivlev/caesarcipher/Main.java

```diff
@@ -0,0 +1,10 @@
+package ru.javarush.ivlev.caesarcipher;
+
+
+public class Main  {
+
+    public static void main(String[] args) {
+        UserDialog dialog  = new UserDialog(System.in, System.out);
+        dialog.startDialog();
+    }
+}
```

Comment:
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log